tutorials/tts/Fastpitch_Training_GermanTTS.ipynb

```diff
@@ -0,0 +1,1112 @@
+    "    input_size: ${model.symbols_embedding_dim}\n",
+    "    kernel_size: 3\n",
+    "    filter_size: 256\n",
+    "    dropout: 0.1\n",
+    "    n_layers: 2\n",
+    "\n",
+    "  pitch_predictor:\n",
+    "    _target_: nemo.collections.tts.modules.fastpitch.TemporalPredictor\n",
+    "    input_size: ${model.symbols_embedding_dim}\n",
+    "    kernel_size: 3\n",
+    "    filter_size: 256\n",
+    "    dropout: 0.1\n",
+    "    n_layers: 2\n",
+    "\n",
+    "  optim:\n",
+    "    name: adamw\n",
+    "    lr: 1e-1\n",
+    "    # optimizer arguments\n",
+    "    betas: [0.9, 0.98]\n",
+    "    weight_decay: 1e-6\n",
+    "\n",
+    "    sched:\n",
+    "      name: NoamAnnealing\n",
+    "      warmup_steps: 1000\n",
+    "      last_epoch: -1\n",
+    "      d_model: 1 # Disable scaling based on model dim\n",
+    "\n",
+    "trainer:\n",
+    "  num_nodes: 1\n",
+    "  devices: -1\n",
+    "  accelerator: gpu\n",
+    "  strategy: ddp\n",
+    "  precision: 32\n",
+    "  max_epochs: 1000\n",
+    "  accumulate_grad_batches: 1\n",
+    "  gradient_clip_val: 1000.0\n",
+    "  checkpoint_callback: False # Provided by exp_manager\n",
+    "  logger: false # Provided by exp_manager\n",
+    "  log_every_n_steps: 100\n",
+    "  flush_logs_every_n_steps: 1000\n",
+    "  check_val_every_n_epoch: 5\n",
+    "\n",
+    "exp_manager:\n",
+    "  exp_dir: ${exp_dir_var}\n",
+    "  name: ${name}\n",
+    "  create_tensorboard_logger: true\n",
+    "  create_checkpoint_callback: true\n",
+    "  checkpoint_callback_params:\n",
+    "    monitor: v_loss\n",
+    "  resume_if_exists: false\n",
+    "  resume_ignore_no_checkpoint: false\n",
+    "\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9fba7bfe",
+   "metadata": {},
+   "source": [
+    "If you are using Weights and Biases, you may need to login first:\n",
+    "```bash\n",
+    "wandb login <api-key>\n",
+    "```\n",
+    "More details [here](https://docs.wandb.ai/ref/cli/wandb-login).\n",
+    "\n",
+    "Now we are ready for training our model! Let's try to train FastPitch.\n",
+    "\n",
+    "```bash\n",
+    "python examples/tts/fastpitch.py --config-path conf/de --config-name fastpitch_align_de \\\n",
+    "    model.train_ds.dataloader_params.batch_size=32 \\\n",
+    "    model.validation_ds.dataloader_params.batch_size=32 \\\n",
+    "    train_dataset=/Data/thorsten-de/train_phonemes_manifest.json \\\n",
+    "    validation_datasets=/Data/thorsten-de/val_phonemes_manifest.json \\\n",
+    "    sup_data_path=/Data/thorsten-de/phonemes/ \\\n",
+    "    exp_dir_var=/result \\\n",
+    "    pitch_mean=132.524658203125 \\\n",
+    "    pitch_std=37.389366149902344 \\\n",
+    "    +exp_manager.create_wandb_logger=true \\\n",
+    "    +exp_manager.wandb_logger_kwargs.name=<wandb_run_name> \\\n",
+    "    +exp_manager.wandb_logger_kwargs.project=<wandb_project_name>\n",
```

Comment:
Please change to be runnable--maybe set `max_epochs` to something low so it can be completed within a small amount of time, and add a disclaimer that it won't fully train the model.

Ideally we would just like users to be able to run it to make sure nothing errors out, and to show that all the previous steps have worked.